test/unit/specs/components/Context/placeContextFactory.spec.js

```diff
@@ -1,32 +1,23 @@
 import placeContextFactory from "../../../../../src/components/Context/placeContextFactory";
 
 describe("Context::placeContextFactory", () => {
-  let window;
   let event;
   let dateProvider;
-  const date = new Date(1553550978123);
+  const date = new Date("March 25, 2019 21:56:18");
 
   beforeEach(() => {
-    window = {
-      screen: {
-        width: 600,
-        height: 800
-      }
-    };
     event = jasmine.createSpyObj("event", ["mergePlaceContext"]);
     dateProvider = () => {
       return date;
     };
   });
 
   it("adds placeContext", () => {
-    placeContextFactory(window, dateProvider)(event);
+    spyOn(date, "getTimezoneOffset").and.returnValue(7 * 60);
+    placeContextFactory(dateProvider)(event);
     expect(event.mergePlaceContext).toHaveBeenCalledWith({
-      localTime: "2019-03-25T21:56:18.123Z",
-      // browsers don't have support for setting the timezone on a date object, because of
-      // this I cannot have hard-coded integer here because depending on the default
-      // timezone of the browser you are testing with you will get different results.
-      localTimezoneOffset: date.getTimezoneOffset()
+      localTime: "2019-03-25T21:56:18-07:00",
+      localTimezoneOffset: 7 * 60
```

Comment:
It seems odd that `localTime` includes the timezone offset, but then we also need to pass the timezone offset in the payload separately. Any idea why this is?

Reply:
I think this make reporting easier.  i.e. if you wanted to do a report broken down by the local hour of the day, you could use localTime and pull out the hour.